src/commands/flinkUDFs.ts

```diff
@@ -121,13 +120,13 @@ export async function startGuidedUdfCreationCommand(selectedArtifact: FlinkArtif
     );
   } catch (err) {
     if (!(err instanceof Error && err.message.includes("Failed to create UDF function"))) {
-      let errorMessage = "Failed to create UDF function: ";
+      let errorMessage = "Failed to create UDF function";
```

Comment:
Adjusted to remove double-spacing (e.g. `Failed to create UDF function:  error message here`) - if we somehow get a non-ResponseError/-Error, we'll just show `Failed to create UDF function` without the colon